### PR TITLE
Allow .co.za and dk renewals

### PR DESCRIPTION
**Pull request by RayeN**
Related tickets https://github.com/yola/production/issues/2040 and https://github.com/yola/production/issues/2972

### CODE REVIEW

File: opensrs/opensrsapi.py [hunk 1]
```diff
@@ -443,11 +444,10 @@ def renew_domain(self, domain, current_expiration_year, period):
...
         except errors.XCPError, e:
-            # We cannot renew .de domains right now (they are
-            # auto-renewed automatically on registar side).
-            # Therefore consider each .de domain renewal
-            # attempt as successful
-            if self._already_renewed(e) or self._is_de_domain_case(e, domain):
+            # We cannot control domains which are automatically renewed on
+            # OpenSRS side. Thus we always treat them as already renewed
+            # for each renewal attempt.
+            if self._already_renewed(e) or self._is_auto_renewed(e, domain):
```
Comment (ekoval) [inline opensrs/opensrsapi.py]: Would it be more efficient if we swap these conditions?

**Comment by ekoval** [comment]
:+1:

File: opensrs/opensrsapi.py (hunk 1 above)
Comment (RayeN) [inline opensrs/opensrsapi.py]: It's a hard question (at least for me) and I dunno the answer for it :(.  We don't have many auto-renewed domains, so, intuitively I suppose `_already_renewed()` will be happening more often.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # [Changelog](https://github.com/yola/opensrs/releases)
 
+## 1.0.0
+* Decided it's stable enough for 1.0.0 [relevant discussion][prod2976]
+* Added more auto-renewed TLDs (.za and .dk) ([#6][6])
+
+[prod2976]: https://github.com/yola/production/issues/2976
```
Comment (adrianmoisey) [inline CHANGELOG.md]: I'm not sure if it makes sense to reference a private repo from a public repo

Comment (RayeN) [inline CHANGELOG.md]: Yeah, my bad, will remove that.

**Comment by adrianmoisey** [comment]
:+1:

**Pull request merged**